packages/opentelemetry-core/src/utils/url.ts

```diff
@@ -14,35 +14,83 @@
  * limitations under the License.
  */
 
-/**
- * Check if {@param url} matches {@param urlToMatch}
- * @param url
- * @param urlToMatch
- */
-export function urlMatches(url: string, urlToMatch: string | RegExp): boolean {
-  if (typeof urlToMatch === 'string') {
-    return url === urlToMatch;
-  } else {
-    return !!url.match(urlToMatch);
-  }
-}
+import { IgnoreMatcher } from "@opentelemetry/types";
+
+// /**
```

Comment:
why is  this commented ?

Reply:
It is dead code that i'm going to delete